openassessment/templates/openassessmentblock/oa_submission_answer.html

```diff
@@ -0,0 +1,17 @@
+{% spaceless %}
+{% load i18n %}
+<ol class="submission__answer__display__content">
+    {% for part in answer.parts %}
+        <li class="submission__answer__part">
+            <article class="submission__answer__part__prompt">
+                {{ part.prompt.description|linebreaks }}
```

Comment:
Add in a context header here (like line 58 in oa_response.html) for screenreaders.